README.md

```diff
@@ -1,2 +1,177 @@
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Microverse Course <a name="about-project"></a>
+
+> This project has two pages. In the home page you can find about teachers, programs, and partners. In the about page you can find about the course.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### HTML, CSS, and JavaScript <a name="HTML and Css"></a>
+
+> HTML, CSS, and JavaScript are tools that web developers use to add text, photos, and videos, and also to design a website
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">Express.js</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> The key features of this application:
+
+- **My informations**
+- **My works**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+[Click here for watching project presentation](https://www.loom.com/share/a7033791d19d43eb9ee28184478c0441)
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+> You can have access to my project online by clicking on the below link:
+
+- [Live Demo Link](https://batoorsohail.github.io/microverse-course/index.html)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+> If you want to use my project you should clone it into your computer and then open it in your code editor.
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+- GitHub 
+- Install Git 
+- Code Editor
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="Sohail Batoor"></a>
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/batoorsohail)
+- Twitter: [@twitterhandle](https://www.linkedin.com/in/sohail-batoor-52429b230/)
+- LinkedIn: [LinkedIn](https://twitter.com/sohailBatoor)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Program**
+- [ ] **Join**
+- [ ] **News**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+This project is the second project that I make. If you like my project please give it a Star.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
```

Comment:
- [ ] Kindly give appropriate credit to [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 🤝️

Reply:
Fixed. But I recommend you to link the name [Cindy Shin](https://www.behance.net/adagio07) the [author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), as required by the [Creative Commons License](https://creativecommons.org/licenses/).